app/(header-default)/erreur/trop-de-demandes/page.tsx

```diff
@@ -1,13 +1,16 @@
-import { GetServerSideProps } from 'next';
 import ButtonLink from '#components-ui/button';
-import Meta from '#components/meta/meta-client';
 import constants from '#models/constants';
-import { NextPageWithLayout } from 'pages/_app';
 
-const TooManyRequest: NextPageWithLayout = () => {
+export async function generateMetadata() {
+  return {
+    title: 'Trop de requête de votre part',
+    robots: { noindex: true },
+  };
+}
+export const tooManyRequestsPathForStatusCode = '/erreur/trop-de-demandes';
```

Comment:
I think I start to undestand, but these regex need a comment, maybe a link to the documentation